components/dropdown/user/src/index.js

```diff
@@ -57,7 +60,10 @@ const DropdownUser = ({
           <span className="sui-DropdownUserMenu-listText">{text}</span>
           {hasLinkNotifications && (
             <span className="sui-DropdownUserMenu-listNotification">
```

Comment:
this span is needed?

Reply:
> this span is needed?

Yes I think so, cause you still need to position the counter within the dropdown (and we can't add classNames to `badgeCounters` wink wink ;)

Reply:
Exactlly, it's as Turo comments.